templates/SilverStripe/View/Shortcodes/EmbedShortcodeProvider_video.ss

```diff
@@ -3,6 +3,6 @@
 >
     {$Content}
     <% if $Arguments.caption %>
-        <p class="caption">{$Arguments.caption}</p>
+        <p class="caption">{$Arguments.caption.RAW}</p>
```

Comment:
This seems wrong, testing locally `something & something` will be rendered as `something &amp; something` - this wasn't happening before this PR

Reply:
@emteknetnz, could you please provide steps to reproduce issue. I don't see any issue.

Reply:
OK ignore this, I think I must have tested on the wrong environment